package.json

```diff
@@ -84,7 +84,10 @@
         ],
         "extends": [
           "plugin:astro/recommended"
-        ]
+        ],
+        "rules": {
+          "prettier/prettier": "off"
+        }
```

Comment:
Without disabling the prettier rules, we get errors like this:
```
  2:2  error  Parsing error: `import` can only be used in `import()` or `import.meta`  prettier/prettier
```

I've tried configuring the `parser` and `parserOptions`, but cannot get the prettier rules to work in conjunction with the the [`eslint-plugin-astro`](https://ota-meshi.github.io/eslint-plugin-astro/) any more (it worked prior to the Astro upgrade).  I'll try to gather details and ticket upstream.